pkg/kubernetes/manifest/manifest.go

```diff
@@ -62,7 +62,7 @@ func (m Manifest) Verify() error {
 		if !o.Get("metadata").IsMSI() {
 			fields["metadata"] = ErrInvalidMap
 		}
-		if !o.Get("metadata.name").IsStr() {
+		if !o.Get("metadata.name").IsStr() && !o.Get("metadata.generateName").IsStr() {
```

Comment:
I don't know this code very well, not sure if we need to improve the error message too. @sh0rez